src/Microsoft.AspNetCore.Authentication/AuthenticationHandler.cs

```diff
@@ -48,6 +48,8 @@ protected HttpResponse Response
         /// </summary>
         protected virtual object Events { get; set; }
 
+        protected virtual string ClaimsIssuer => Options.ClaimsIssuer ?? Scheme.Name;
```

Comment:
So we have to decide whether we require all AddXyz's to chaining to a new AddAuthenticationHandler method which adds an Initializer that just defaults ClaimsIssuer to scheme.Name, or add this new property on the handler which does the defaulting.

I'm leaning towards the property.  But we still are going to have chaining for each virtual level of InitializeOptions we have today, so there's going to be an AddRemoteAuthenticationHandler in the next iteration.